### PR TITLE
Fix for #39

### PR DESCRIPTION
**Pull request by AgarFu**
## Pull Request Checklist

This PR fixes issue #39 

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [X] Binstubs are created if needed

- [X] RuboCop passes

- [X] Existing tests pass 

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [3.0.1] - 2018-03-30
```
Comment (majormoses) [inline CHANGELOG.md]: Please avoid versioning the changes in a PR per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) there are several reasons for this:
- PRs are not a FIFO queue, merge order is not guaranteed
- maintainer may disagree with you (in this case its a patch)
- we date the release not the contribution submitted, there is no SLA on review will always happen same day submitted
- attempt to reduce the number of time needing to rebase

File: lib/sensu-plugins-kubernetes/version.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module SensuPluginsKubernetes
   module Version
     MAJOR = 3
     MINOR = 0
-    PATCH = 0
+    PATCH = 1
```
Comment (majormoses) [inline lib/sensu-plugins-kubernetes/version.rb]: please avoid versioning in PRs. I will accept it this time.

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

**Comment by AgarFu** [comment]
I'll create a test for it, I'm happy reverting the changelog changes

**Comment by AgarFu** [comment]
@khilawar4

Please take a look at this bugfix.

**Comment by majormoses** [comment]
I am currently on vacation and will review this again when I have some time.

**Comment by majormoses** [comment]
Sorry for the long delay I was on vaction, started a new job and had other things going on as well so this dropped off my radar.

Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

**Comment by majormoses** [comment]
Sorry if I was not clear we actually want you to include the changelog entry just not version and date it, it should remain under `### [Unreleased]` I will add that back in before releasing.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-kubernetes/versions/3.0.1

**Comment by AgarFu** [comment]
Awesome, thx!

**Pull request merged**